my-app/src/app/screens/Game/index.js

```diff
@@ -1,47 +1,24 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 
 import calculateWinner from '../../utils/utils';
+import { actions } from '../../../redux/game/actions';
 
 import Board from './components/Board';
 
 class Game extends Component {
-  state = {
-    history: [{ squares: Array(9).fill(null) }],
-    xIsNext: true,
-    stepNumber: 0
-  };
-
   handleClick = i => {
-    const history = this.state.history.slice(0, this.state.stepNumber + 1);
-    const current = history[history.length - 1];
-    const squares = current.squares.slice();
-    if (calculateWinner(squares) || squares[i]) {
-      return;
-    }
-    squares[i] = this.state.xIsNext ? 'X' : 'O';
-    this.setState({
-      history: history.concat([
-        {
-          squares: squares
-        }
-      ]),
-      stepNumber: history.length,
-      xIsNext: !this.state.xIsNext
-    });
+    this.props.dispatch({ type: actions.handleClick, pos: i });
```

Comment:
I'd extract all actions into an `actionCreators` object placed at `actions.js`, so all actions forms keep centralized in a single file. What do you think ?

Reply:
A convention is to use a `payload` for all relevant data that the reducer needs to perform operations. By example:
```
{ type: actions.handleClick, payload: { pos: i } }
```
It's a bit less practical but libraries like redux-recompose approaches this convention to simplify code

Reply:
I think we may receive this `handleClick` via `mapDispatchToProps`